tests/functional/adapter/test_basic.py

```diff
@@ -13,52 +14,188 @@
     pass
 
 
 class TestSingularTestsEphemeralTrino(BaseSingularTestsEphemeral):
-    pass
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "name": "singular_tests_ephemeral",
+            "seeds": {
+                "+column_types": {"some_date": "timestamp(6)"},
+            },
+        }
+
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {
+            "base.csv": seeds_base_csv,
+        }
 
 
 class TestEmptyTrino(BaseEmpty):
     pass
 
 
 class TestEphemeralTrino(BaseEphemeral):
-    pass
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "name": "ephemeral",
+            "seeds": {
+                "+column_types": {"some_date": "timestamp(6)"},
+            },
+        }
+
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {
+            "base.csv": seeds_base_csv,
+        }
 
 
 class TestIncrementalTrino(BaseIncremental):
-    pass
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "name": "incremental",
+            "seeds": {
+                "+column_types": {"some_date": "timestamp(6)"},
+            },
+        }
+
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {"base.csv": seeds_base_csv, "added.csv": seeds_added_csv}
 
 
 class TestGenericTestsTrino(BaseGenericTests):
-    pass
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "name": "generic_tests",
+            "seeds": {
+                "+column_types": {"some_date": "timestamp(6)"},
+            },
+        }
+
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {"base.csv": seeds_base_csv, "schema.yml": generic_test_seed_yml}
 
 
 class TestTrinoValidateConnection(BaseValidateConnection):
     pass
 
 
 class TestDocsGenerateTrino(BaseDocsGenerate):
+    @pytest.fixture(scope="class")
+    def project_config_update(self, unique_schema):
+        alternate_schema = unique_schema + "_test"
+        return {
+            "asset-paths": ["assets", "invalid-asset-paths"],
+            "vars": {
+                "test_schema": unique_schema,
+                "alternate_schema": alternate_schema,
+            },
+            "seeds": {
+                "quote_columns": True,
+                "+column_types": {"updated_at": "timestamp(6)"},
+            },
+            "quoting": {"identifier": False},
+        }
+
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {"schema.yml": seed__schema_yml, "seed.csv": seed__seed_csv}
+
     @pytest.fixture(scope="class")
     def expected_catalog(self, project, profile_user):
         return base_expected_catalog(
             project,
             role=None,
             id_type="integer",
             text_type="varchar",
-            time_type="timestamp(3)",
+            time_type="timestamp(6)",
```

Comment:
Cause timestamp(6) is expected here since we convert it here: https://github.com/starburstdata/dbt-trino/pull/123/files#diff-241565d7b2babaa70d14225f4e4860a9c244a13840f94ca40345a6d8b68ba56cR182